bizdemo/easy_note/README.md

```diff
@@ -142,7 +142,7 @@ curl --location --request POST '127.0.0.1:8080/v1/user/login' \
 #### response
 ```javascript
 {
-    "code": 200,
+    "code": 0,
```

Comment:
Would it be better to improve the readme description and add other cases?

Reply:
I got it, thx